db/migrate/20190509014848_create_clearstream_msg_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateClearstreamMsgEvents < ActiveRecord::Migration[6.0]
+  def change
+    #create_table :clearstream_msg_events, id: :uuid do |t|
+    create_table :clearstream_msg_events do |t|
```

Comment:
why not using uuid here?

Reply:
Don't see the need.  The only time a ClearstreamMsgEvent record will be created is in a Sidekiq worker, e.g. UpdateClearstreamMsgStatusFromWebhookWorker.  Currently, the only time it's ID will be viewable is in the admin app.  I'm also unaware of any need for sharding in the future.  Do you see a need to change it from bigint to uuid?

I see that either I need to put it back (if you see a reason to do so) or I need to change the id type for sendgrid_msg_events.